modules/c/nitf/source/Reader.c

```diff
@@ -511,12 +511,17 @@ NITFPRIV(NITF_BOOL) readImageSubheader(nitf_Reader * reader,
 {
     unsigned int i;
     nitf_ImageSubheader *subhdr;
+    nitf_FileHeader *hdr = reader->record->header;
     /* List iterator pointing to the image segment */
     nitf_ListIterator listIter = nitf_List_begin(reader->record->images);
 
     nitf_Uint32 numComments;    /* Number of comment fields */
     nitf_Uint32 nbands;         /* An integer representing the \nbands field */
     nitf_Uint32 xbands;         /* An integer representing the xbands field */
+    nitf_Uint32 imageSubheaderStart;  /* Start position of image subheader */
```

Comment:
Not a huge deal but it looks like these variable names don't match the DESubheader variable names?
Should these be called startOffset and endOffset as well?

Reply:
Good point.